File: src/C#/StructuralPatterns/Composite/WindowsFormsApplication/Composites/Panel.cs
using System.Drawing;
using WindowsFormsApplication.Composites.Abstraction;

namespace WindowsFormsApplication.Composites
{
    public class Panel : BaseControlComposite
    {
        public Panel(string name, int width, int height, Color foreColor, Color backColor)
            : base(name, width, height, foreColor, backColor)
        {

        }
    }
}

File: src/PHP/StructuralPatterns/Composite/WindowsFormsApplication/Composites/Panel.php
<?php

namespace StructuralPatterns\Composite\WindowsFormsApplication\Composites;

class Panel extends BaseControlComposite
{
    public function __construct($name, $width, $height, $foreColor, $backColor)
    {
        parent::__construct($name, $width, $height, $foreColor, $backColor);
    }
}